cmake/Options.cmake

```diff
@@ -48,3 +48,5 @@ option(TP_BUILD_LIBUV "Build libuv from source" OFF)
 include(GNUInstallDirs)
 set(TP_INSTALL_LIBDIR ${CMAKE_INSTALL_LIBDIR} CACHE STRING "Directory in which to install libraries")
 mark_as_advanced(TP_INSTALL_LIBDIR)
+set(TP_INSTALL_INCLUDEDIR ${CMAKE_INSTALL_INCLUDEDIR} CACHE STRING "Directory in which to install public headers")
+mark_as_advanced(TP_INSTALL_INCLUDEDIR)
```

Comment:
Is it something you want to keep? It's quite unusual.

Reply:
I must admit I don't remember why we have it. Is there harm in keeping it? What's the "preferred" way?

Reply:
It doesn't harm I think. If consumer wants to change the standard install layout (bin, lib, include, share folders), I guess it's useful, but uncommon.